packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -4028,6 +4023,13 @@ export class ContainerRuntime
 				});
 			}
 
+			const serializedContent = JSON.stringify(containerRuntimeMessage);
+			const message: BatchMessage = {
+				contents: serializedContent,
+				metadata,
+				localOpMetadata,
+				referenceSequenceNumber: this.deltaManager.lastSequenceNumber,
+			};
```

Comment:
There is no functionality change here (atleast intentional). The reason I am moving the code here is because in general JS  convention is to declare variables closer to usage - in this case, `message`

Reply:
While you're here, consider renaming `const contents = this.documentsSchemaController.maybeSendSchemaMessage();` to `documentSchemaChangeMessage` or something?